packages/carousel/src/styledComponents/Carousel.js

```diff
@@ -0,0 +1,42 @@
+import styled from "styled-components";
+import { colors } from "@crave/farmblocks-theme";
+
+const scale = 1.219;
+
+const Container = styled.div`
+  position: relative;
+  ul {
+    position: relative;
+    left: 50%;
+    margin-left: ${props => -props.itemConfig.width * scale / 2}px;
```

Comment:
some parenthesis would be nice to make it better to read :P

`margin-left: ${props => - (props.itemConfig.width * scale / 2)}px;`

or even `margin-left: ${props => -1 * props.itemConfig.width * scale / 2}px;`

Also is it possible to leave some comments explaining these formulas? In the future we will forget why we used it this way